main-command/src/main/scala/sbt/State.scala

```diff
@@ -233,8 +233,11 @@ object State {
     )
   }
 
+  @deprecated("Import State._ or State.StateOpsImpl to access state extension methods", "1.3.0")
+  def stateOps(s: State): StateOps = new StateOpsImpl(s)
```

Comment:
Does this break source compatibility if someone's build.sbt had `"compile" :: state`?

Reply:
No, because under the hood an implicit class is implemented with an implicit def. But to be double check, I did add `commands += Command.command("alternative-compile") { s => "compile" :: s }` to a simple build.sbt and I had no problems loading the build with my snapshot sbt version.

Reply:
Could've also just called the class `stateOps` 😉

Reply:
I think it's not too late!

Reply:
Oh yeah. 😄 https://github.com/sbt/sbt/pull/4455